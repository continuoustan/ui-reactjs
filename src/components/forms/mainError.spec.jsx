import React from 'react';
import renderer from 'react-test-renderer';

import MainError from './mainError';

test('Main Error Display', () => {
    const error = {};
    const tree = renderer.create(
        <MainError error={error} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
