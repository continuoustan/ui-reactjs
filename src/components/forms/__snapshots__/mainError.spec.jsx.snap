// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Error Display 1`] = `null`;
